Angular 1.4.7 fix filter:notarray

lukaszpaczos opened the pull request:
Connected with this issue:
https://github.com/mgcrea/angular-strap/issues/1567

When I switch from angular 1.3.15 to 1.4.7 i saw that i got error like this:

Error: [filter:notarray] Expected array but received: false

So i propose to add this simple fix to avoid it.

Best regards

vmlf01 commented:
Do you have a sample with the problem you could share? Why would the array variable not be an array? Just by setting it to an empty array you might avoid the error but cause other side-effects (what was it set to before?).

lukaszpaczos commented:
You have right...I don't realize that I has "return false" instead of "return []" and that was the reason why I got error.

I fixed this because I thought was angular-strap problem, but in new version of angular (I changed from 1.3.15 to 1.4.7) they start to check in filter that the variable is array, so when on old version of angular everything working well I thought that maybe something wrong with angular-strap :)

So fix is not needed :)

Cheers
